meteor/settings.py

```diff
@@ -2,3 +2,4 @@
 TV_STOP_TOLERANCE: float = 0.00000005
 TV_MAX_NUM_ITER: int = 50
 TV_MAP_SAMPLING: int = 3
+KWEIGHTED_MAP_SAMPLING: int = 4
```

Comment:
hmmm why? i guess we need to figure out all this sampling stuff

Reply:
This actually is a slip. I think we should pick one sampling and stick with it... tradeoff between compute and quality I guess?